ambari-server/src/main/java/org/apache/ambari/server/audit/event/request/ViewPrivilegeChangeRequestAuditEvent.java

```diff
@@ -119,9 +117,7 @@ protected void buildAuditMessage(StringBuilder builder) {
           lines.add("  Roles: " + StringUtils.join(roles.get(role), ", "));
         }
       }
-
-      builder.append(StringUtils.join(lines, System.lineSeparator()));
-
+      builder.append(StringUtils.join(lines, " , "));
```

Comment:
the separator should be `;` since `,` is used to separate the values for each entry.